paper-drawer-panel.html

```diff
@@ -393,6 +396,22 @@
         this.transition = true;
       },
 
+      attached: function() {
+        this.listen(this.$.main, 'transitionend', '_transitionEndCallback');
+        this.listen(this.$.main, 'webkitTransitionEnd', '_transitionEndCallback');
+      },
+
+      detached: function() {
+        this.unlisten(this.$.main, 'transitionend', '_transitionEndCallback');
+        this.unlisten(this.$.main, 'webkitTransitionEnd', '_transitionEndCallback');
+      },
+
+      _transitionEndCallback: function (e) {
```

Comment:
Can you change the method name to something like `_onMainTransitionEnd` so we know is for the transitionEnd on the #main?